libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -314,10 +314,8 @@ void YamlConfiguration::parseProvenanceReportingYaml(YAML::Node *reportNode, cor
   auto schedulingStrategyStr = node["scheduling strategy"].as<std::string>();
   checkRequiredField(&node, "scheduling period", CONFIG_YAML_PROVENANCE_REPORT_KEY);
   auto schedulingPeriodStr = node["scheduling period"].as<std::string>();
-  checkRequiredField(&node, "host", CONFIG_YAML_PROVENANCE_REPORT_KEY);
-  auto hostStr = node["host"].as<std::string>();
-  checkRequiredField(&node, "port", CONFIG_YAML_PROVENANCE_REPORT_KEY);
-  auto portStr = node["port"].as<std::string>();
+  checkRequiredField(&node, "url", CONFIG_YAML_PROVENANCE_REPORT_KEY);
```

Comment:
If we tagged today would people who have tried MiNiFi in cpp 0.2.0 be able to use the same config files in cpp 0.3.0?